src/lib.rs

```diff
@@ -1214,6 +1214,7 @@ pub enum SampleLevel {
         x: Handle<Expression>,
         y: Handle<Expression>,
     },
+    Base,
```

Comment:
The spec refers to mip level 0 as the base level but we use `Zero` in the IR.
I'd add a `clamp_to_edge` bool on `Expression::ImageSample` instead.